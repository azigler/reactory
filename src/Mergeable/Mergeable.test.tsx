import React from "react"
import { createRoot } from "react-dom/client"
import { act } from "react-dom/test-utils"
import { Mergeable } from ".."

describe("Draggable component", () => {
  it("renders", () => {
    const container = document.createElement("main")
    const root = createRoot(container)
    act(() => {
      root.render(<Mergeable />)
    })
    expect(container).toMatchSnapshot()
  })
})
