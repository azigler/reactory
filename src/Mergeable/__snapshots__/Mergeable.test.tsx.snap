// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Draggable component renders 1`] = `
<main>
  <div
    class="react-draggable"
    style="transform: translate(0px,0px);"
  >
    BOOP
  </div>
</main>
`;
